README.md

```diff
@@ -124,7 +124,7 @@ repositories {
   mavenCentral()
 }
 dependencies {
-  implementation 'app.cash.quickjs:quickjs-android:0.9.2'
+  implementation 'app.cash.zipline:zipline-android:0.9.2'
```

Comment:
I would not update the README and let it track the latest release until closer to the first zipline release. These Maven coordinates were already broken after the switch to multiplatform, for example, but I was going to wait until release to fix them.